src/libNode/Node.h

```diff
@@ -155,8 +155,8 @@ class Node : public Executable, public Broadcastable
 
     // internal calls from ProcessStartPoW1
     bool ReadVariablesFromStartPoWMessage(
-        const vector<unsigned char>& message, unsigned int offset,
-        uint64_t& block_num, uint8_t& dsDifficulty, uint8_t& difficulty,
+        const vector<unsigned char>& message, unsigned int cur_offset,
+        uint64_t& block_num, uint8_t& ds_difficulty, uint8_t& difficulty,
```

Comment:
Although it is automated, I prefer it to be without _ for consistency with other code. same for block_num.

Reply:
The name with `_` is taken from the definition of this function. It didn't invent this name here.

Reply:
maybe we can do a minor clean up on these few cases. If not, we can also fix these kind of issues in future pr. I let you make the call for this.

Reply:
perhaps let me find some checker tool for the naming of parameters and variables.